bigtable-dataflow-parent/bigtable-beam-import/src/main/java/com/google/cloud/bigtable/beam/TemplateUtils.java

```diff
@@ -45,6 +46,14 @@ public static CloudBigtableTableConfiguration BuildImportConfig(ImportOptions op
     if (opts.getBigtableAppProfileId() != null) {
       builder.withAppProfileId(opts.getBigtableAppProfileId());
     }
+
+    if (opts.getBigtableWriteThrottleMs() != null) {
```

Comment:
`getBigtableWriteThrottleMs()` returns a ValueProvider which will always be non-null. I think it would be more correct to use a `NestedValueProvider` that transforms the `opts.getBigtableWriteThrottleMs()` `ValueProvider` into a "true" or "false" for the `google.bigtable.buffered.mutator.throttling.enable` value

Reply:
Should be something like this:

ValueProvier<Boolean> enableThrottling = NestedValueProvider.of(
        options.getBigtableWriteThrottleMs(),
        new SerializableFunction<Integer, Boolean>() {
          @Override
          public Boolean apply(Integer throttleMs ) {
            return throttleMs > 0;
          }
        });

builder.withConfiguration(BigtableOptionsFactory.BIGTABLE_BUFFERED_MUTATOR_ENABLE_THROTTLING, enableThrottling);
builder.withConfiguration(BigTableOptionsFactory.BIGTABLE_BUFFERED_MUTATOR_THROTTLING_THRESHOLD_MILLIS, options.getBigtableWriteThrottleMs());